features/gip/lb.pan

```diff
@@ -20,10 +20,9 @@ variable GIP_PROVIDER_WRAPPER_LB ?= 'glite-info-service-lbserver';
     SELF['confFiles'][escape(GIP_PROVIDER_SERVICE_CONF_LB)] = 
         "init = "+GIP_PROVIDER_SERVICE_INIT_LB+"\n" +
         "service_type = "+GIP_PROVIDER_SERVICE_TYPE_LB+"\n" +
-        "get_version = rpm -q  glite-lb-ws-interface --queryformat '%{version}\\n'\n" +
+        "get_version = rpm -qa | grep 'glite-lb-server-[0-9]' | cut -d- -f4\n" +
```

Comment:
Isn't this highly reliant on the output format of `rpm`?

Wouldn't `rpm -q glite-lb-server --queryformat '%{version}\\n'` be safer and require less pipes?